gst/datarepo/gstdatareposrc.h

```diff
@@ -30,6 +30,9 @@ G_BEGIN_DECLS
 #define GST_IS_DATA_REPO_SRC_CLASS(klass) \
   (G_TYPE_CHECK_CLASS_TYPE((klass),GST_TYPE_DATA_REPO_SRC))
 
+/* media_type has not IMAGE type */
+#define _NNS_IMAGE 5  /**<supposedly image/png, image/jpeg and etc */
```

Comment:
TODO: future work, let's consider adding new enum of media type for datarepo elements.
nns_media_type was supposed converting media input to tensor, but datarepo may have various data type (e.g. video format yub). So I think it is better to define media type in datarepo elements, or just declare caps-any to handle all data format later.

Reply:
okay, thank you.